nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/src/main/java/org/apache/nifi/util/hive/HiveConfigurator.java

```diff
@@ -71,6 +72,14 @@ public Configuration getConfigurationFromFiles(final String configFiles) {
         return hiveConfig;
     }
 
+    public void preload(Configuration configuration) {
+        try {
+            FileSystem.get(configuration);
```

Comment:
This causes the unit tests to fail, not because of your code but because of the invalid value in the test hive-site.xml. I will update the file before merging your PR.